contract/r/gnoswap/gov/governance/execute_test.gno

```diff
@@ -367,3 +368,103 @@ func TestParameterRegistry2(t *testing.T) {
 		})
 	}
 }
+
+func TestAtomicExecution(t *testing.T) {
```

Comment:
When I tested it without validation fix, it's also passed. I think we need to make a new testcase to raise the validation failed error

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/617/commits/b75c2991e959d78e7bcc48d3f66e2d4371138cf2